src/explore-education-statistics-common/src/components/Code.module.scss

```diff
@@ -0,0 +1,36 @@
+@import '~govuk-frontend/dist/govuk/base';
+
+.container {
+  position: relative;
+
+  &:active.copyButton,
+  &:focus-within.copyButton,
+  &:hover.copyButton {
+    opacity: 1;
+  }
+}
+
+.copyButton {
+  max-width: fit-content;
+  position: absolute;
+  right: govuk-spacing(2);
+  top: govuk-spacing(2);
+
+  &:active {
+    top: 12px;
+  }
+}
+
+.pre {
+  border: 1px solid #bfc1c3;
+  background: #f9f9f8;
+  max-width: 100%;
+  overflow-x: auto;
+  padding: 60px 15px 15px !important;
+
+  &:focus {
+    border: $govuk-focus-width solid #000;
+    outline: $govuk-focus-colour solid $govuk-focus-width;
+    padding: 58px 13px 13px !important; // counter border width to avoid content shift
```

Comment:
Can `box-sizing: border-box` be used to counteract content shift?

Reply:
Great suggestion, but I've since removed this line after removing the ability to focus code snippets